packages/core/src/core/llm-client.ts

```
@@ -141,23 +400,112 @@
+              content: "\n\nProvide response in JSON format.",
+            },
+            {
+              role: "user",
+              content: prompt,
+            },
+          ],
+          maxTokens,
+        });
+        break;
+
+      case "mistral":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)